core/src/main/java/com/radixdlt/rev2/REv2StateComputer.java

```diff
@@ -283,15 +284,15 @@ public StateComputerLedger.StateComputerPrepareResult prepare(
 
   @Override
   public LedgerProofBundle commit(
-      LedgerExtension ledgerExtension, Option<byte[]> serializedVertexStoreState) {
+      LedgerExtension ledgerExtension, Option<WrappedByteArray> serializedVertexStoreState) {
     final var proof = ledgerExtension.proof();
     final var header = proof.ledgerHeader();
 
     var commitRequest =
         new CommitRequest(
             ledgerExtension.transactions(),
             proof,
-            serializedVertexStoreState,
+            serializedVertexStoreState.map(WrappedByteArray::value),
```

Comment:
We should just always handle it as a `WrappedByteArray` I think? It's just safer than passing around a big `byte[]`.

If we need to, let's add an SborCodec for `WrappedByteArray`:

```
  public static void registerCodec(CodecMap codecMap) {
    codecMap.register(
        WrappedByteArray.class,
        codecs ->
            new CustomByteArrayCodec<>(
                WrappedByteArray::value,
                WrappedByteArray::new));
  }
```